### PR TITLE
Template, setting and template tag conversions to Jinja

### PR DESCRIPTION
**Pull request by dan-gamble**
Combined with https://github.com/onespacemedia/project-template/pull/47

### CODE REVIEW

File: cms/apps/pages/templatetags/pages.py [hunk 1]
```diff
@@ -111,17 +118,21 @@ def meta_description(context, description=None):
     """
     if description is None:
         description = context.get("meta_description")
+
+    # TODO: Check in the context for objects for every templatetag like this
```
Comment (danielsamuels) [inline cms/apps/pages/templatetags/pages.py]: Is it worth fixing this TODO before we merge?

File: cms/apps/pages/templatetags/pages.py [hunk 2]
```diff
@@ -7,22 +7,15 @@
...
-def navigation(context, pages, section=None):
-    """
-    Renders a navigation list for the given pages.
-
-    The pages should all be a subclass of PageBase, and possess a get_absolute_url() method.
-
-    You can also specify an alias for the navigation, at which point it will be set in the
-    context rather than rendered.
-    """
+def navigation_entries(context, pages, section=None, is_json=False):
```
Comment (danielsamuels) [inline cms/apps/pages/templatetags/pages.py]: Should this be `get_ navigation_entries`?

Comment (dan-gamble) [inline cms/apps/pages/templatetags/pages.py]: This is a function that is just used in 2 templatetags and isn't actually used in templates. 

https://github.com/onespacemedia/cms/pull/111/files/9bd656b6eaac2db7de6214705b431eef8ed528b5#diff-9baa15ecda158c9c851d5a7b6747a500R61

For example uses it. We could prefix it with `get_` but that kind of conflicts with template based functions that use the `get_` prefix

File: cms/apps/pages/templatetags/pages.py (hunk 1 above)
Comment (dan-gamble) [inline cms/apps/pages/templatetags/pages.py]: We can do, i didn't do it here as it was "out of scope" for a Jinja change. I was gonna do it in a PR after, but i don't mind doing it in this PR.

File: cms/apps/pages/templatetags/pages.py (hunk 2 above)
Comment (danielsamuels) [inline cms/apps/pages/templatetags/pages.py]: What about prefixing with just an underscore to mark it as an internal function?

File: cms/apps/pages/templatetags/pages.py (hunk 1 above)
Comment (danielsamuels) [inline cms/apps/pages/templatetags/pages.py]: That's fair enough, better to keep the scope narrow.

File: cms/apps/pages/templatetags/pages.py (hunk 2 above)
Comment (dan-gamble) [inline cms/apps/pages/templatetags/pages.py]: Sounds good to me 👍

**Pull request merged**